### PR TITLE
add refseq option parameter to variant simulator script

### PR DESCRIPTION
**Pull request by ima23**

### CODE REVIEW

File: tools/variant_simulator/simulate_variation [hunk 1]
```diff
@@ -77,9 +78,13 @@ print "getting species ($species)";
 print ", chrom ($chromOnly)" if $chromOnly;
 print ", gene ($gene)", if $gene;
 print ", out: $outFile\n";
+my $dbtype = "core";
+$dbtype = "otherfeatures" if $refSeq_genes;
```
Comment (ens-lgil) [inline tools/variant_simulator/simulate_variation]: Could we use a more condensed syntax for it ?
``` $dbtype = $refSeq_genes ?  "otherfeatures" : "core";```

File: tools/variant_simulator/simulate_variation [hunk 2]
```diff
@@ -77,9 +78,13 @@ print "getting species ($species)";
...
+my $logic_name;
+$logic_name = "refseq_import" if $refSeq_genes;
```
Comment (ens-lgil) [inline tools/variant_simulator/simulate_variation]: Same as above:
```$logic_name = $refSeq_genes ? "refseq_import" : undef;```

File: tools/variant_simulator/simulate_variation [hunk 3]
```diff
@@ -49,6 +49,7 @@ GetOptions( "species=s" => \$species,
     "codingOnly" => \$codingOnly,
     "intronsOnly" => \$intronOnly,
     "assembly=s" => \$assembly,
+    "refseq"     => \$refSeq_genes,
```
Comment (ens-lgil) [inline tools/variant_simulator/simulate_variation]: For consistency, could we use the same case for `refseq` and `$refSeq_genes` ?
Usually `refseq` is written in Ensembl in lower case.

Comment (ima23) [reply tools/variant_simulator/simulate_variation]: Good point! I've updated this in the next commit.

File: tools/variant_simulator/simulate_variation (hunk 1 above)
Comment (ima23) [reply tools/variant_simulator/simulate_variation]: Thanks for the suggestion! I've updated this in the next commit.

File: tools/variant_simulator/simulate_variation (hunk 2 above)
Comment (ima23) [reply tools/variant_simulator/simulate_variation]: Now updated.

**Pull request merged**